charts/argo-cd/Chart.yaml

```diff
@@ -25,4 +25,4 @@ dependencies:
   - name: argocd-image-updater
     version: 0.1.0
     repository: https://charts.akuity.io
-    condition: imageUpdater.enabled
```

Comment:
We also need to bump argo-cd chart version to `version: 1.1.8`

Reply:
ok done 👍